docs/Tutorial.md

```diff
@@ -372,11 +372,11 @@ In react-admin, fields are just React components. When rendered, they grab the `
 
 That means that you can do the same to write a custom Field. For instance, here is a simplified version of the `<UrlField>`:
 
-```jsx
+```tsx
 // in src/MyUrlField.tsx
 import { useRecordContext } from "react-admin";
 
-const MyUrlField = ({ source }) => {
+const MyUrlField = ({ source }: { source: string }) => {
```

Comment:
You can import `FieldProps` from `react-admin` and use it here:
```suggestion
const MyUrlField = ({ source }: FieldProps) => {
```

Reply:
You marked this as fixed but it isn't.

Reply:
As discussed with @djhi, typed params with `FieldProps` generates Typescript errors, so we decided to let it as is

Reply:
I'll work on a fix when I can